server.js

```diff
@@ -1,7 +1,30 @@
-var express = require('express');
-var app = express();
-app.get('/ping', (req, res) =>{
-    res.send('hello ')
-})
-
-var server = app.listen(3000);
+const express = require("express");
+const app = express();
+const mongoose = require('mongoose');
+
+const uri = "mongodb+srv://hanshul:babaji@cluster0.e06nfsc.mongodb.net/?retryWrites=true&w=majority";
+
+app.get('/', (req, res) => {

```

Comment:
The MongoDB connection is being established on every GET request to the root route. This could lead to performance issues. It's better to establish the connection when the server starts. [important]